AsyncDisplayKit/ASCollectionView.h

```diff
@@ -43,6 +44,7 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (instancetype)initWithCollectionViewLayout:(UICollectionViewLayout *)layout;
 - (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout;
+- (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout layoutFacilitator:(id<ASCollectionViewLayoutFacilitatorProtocol>)layoutFacilitator;
```

Comment:
@binl can you add a similar initializer to ASCollectionNode?  I think those leave out the frame, but we should have one with the layout and layoutFacilitator.  Going forward we should always create Collection / Table Nodes rather than Views, as the direct allocation of the Views will be deprecated in 2.0.